.readthedocs.yaml

```diff
@@ -1,7 +0,0 @@
-version: 2
```

Comment:
We had this to ensure the RTD pages redirect properly IIRC. Does that still work?

The RTD link is: https://conda-forge.readthedocs.io/en/latest/

Reply:
Ah, I didn't know we had that! Sorry and thanks!